src/drawings.jl

```diff
@@ -614,13 +614,164 @@ function Drawing(paper_size::AbstractString, f="luxor-drawing.png"; strokescale=
   Drawing(w, h, f, strokescale=strokescale)
 end
 
+
+@doc raw"""
+    adjust_background_rects(buffer::UInt8[])
+
+See issue  https://github.com/JuliaGraphics/Luxor.jl/issues/150 for discussion details.\
+
+Setting backgrounds in a recording surface (:rec) and creating a svg from it result in elements as
```

Comment:
The idea is to be conservative and doing it only for :rec when written to svg. If some tweak is needed for other formats it can be easily (or not so easy) expanded.